examples/guides/data-driven-viz-1/step-2.html

```diff
@@ -2,16 +2,16 @@
 <html>
 
 <head>
-    <title>Implicit casting | CARTO</title>
+    <title>Implicit casts | CARTO</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
     <!-- Include CARTO VL JS -->
-    <script src="../../../dist/carto-vl.min.js"></script>
+    <script src="https://libs.cartocdn.com/carto-vl/branches/971-guides-aggregation/carto-vl.min.js"></script>
```

Comment:
Hey, this is not good, this links to a CVL bundle of a branch, to a patched MGL version and to a carto.com CSS

Reply:
🤔

I already fixed similar cases when I merged master into this branch in the previous PR, see https://github.com/CartoDB/carto-vl/commit/2ea69c20663953b27e3a0b5c9effd519bb170e2e.

Reply:
Thx david, I'll fix them. I'm checking all the links